docs/DeveloperGuide.md

```diff
@@ -376,6 +376,51 @@ Suppose we delete a single `Tag`, the parent-tags of `Tag` will be reconnected t
 ![single-tag-delete](images/DeleteSingleTagPic.png) <br>
 We have intentionally chosen this design in order to preserve the effectiveness of top-down queries of all sub-tags.
 
+### Event and attendees management
```

Comment:
can you check if there's a hanging ` somewhere? seems a bit odd that the text is blue

Reply:
found it
![image](https://user-images.githubusercontent.com/69659433/98553910-162a8680-22db-11eb-958e-7270318cc998.png)